packages/cli/src/lib/cmds/play.ts

```diff
@@ -50,62 +50,52 @@ class Play {
   * @return {number} The largest version number found in the play console.
   */
   async getLargestVersion(twaManifest: TwaManifest): Promise<number> {
-    // TODO(@nohe427): This will return 0 until refactor finished.
-    const versionNumber = await this.googlePlay.getLargestVersionCode(twaManifest.packageId);
-    return versionNumber;
+    return await this.googlePlay.getLargestVersionCode(twaManifest.packageId);
   }
 
   /**
   * Publishes the Android App Bundle to the specified from the {@link PlayArgs}.
```

Comment:
nit: there's a typo in this line - to the specified what?

Reply:
Think I fixed this